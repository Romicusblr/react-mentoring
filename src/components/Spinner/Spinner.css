.spinner {
  background-color: var(--color-raised);
  width: 5rem;
  height: 5rem;
  border-radius: 50%;
  opacity: 0;
  animation: grow 1s linear infinite;
}

.spinner-wrapper {
  display: flex;
  align-items: center;
  justify-content: center;
}

@keyframes grow {
  0% {
      -webkit-transform: scale(0);
      transform: scale(0);
  }
  50% {
      opacity: 1;
  }
}
