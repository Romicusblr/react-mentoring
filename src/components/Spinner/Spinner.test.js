import Spinner from '.';

describe('Spinner', () => {
  it('should match to snapshot', () => {
    const snap = shallow(<Spinner />);
    expect(snap).toMatchSnapshot();
  });
});
