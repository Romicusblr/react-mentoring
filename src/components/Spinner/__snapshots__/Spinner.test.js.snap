// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner should match to snapshot 1`] = `
<div
  className="spinner-wrapper"
>
  <div
    className="spinner"
  />
</div>
`;
